build.gradle.kts

```diff
@@ -0,0 +1,115 @@
+/*
```

Comment:
Jeg tenker fortsatt denne kan utgå fra sak sin build.gradle.kts og legge til det som mangler. 
Det eneste som burde mangle er avhengighetene til familie

* I denne er det mange versjoner som ikke trenges å settes
* Og noen dependencies som trengs, men som styres av spring management
* `java-library` trengs ikke
* `maven-publish` trengs ikke

Mangler
* failOnNonReproducibleResolution stopper snapshots fra å bli deployet
* ktlint

Reply:
> Jeg tenker fortsatt denne kan utgå fra sak sin build.gradle.kts og legge til det som mangler. Vi får då mer likt oppsett tvers applikasjonene.
Enig i det, men det kan også gjøres i påfølgende oppgaver. I første rekke handla dette om å få til å bygge og deploye på Gradle.

Reply:
Sånn, har gjort den så nær ts-sak sin som mulig. Enkelte avhengigheter måtte jeg spesifisere eksplisitt for at ting skulle fungere korrekt.